sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/LogicalPlan.scala

```diff
@@ -381,6 +381,15 @@ object LogicalPlanIntegrity {
     }.flatten
   }
 
+  def validateSchemaOutput(previousPlan: LogicalPlan, currentPlan: LogicalPlan): Option[String] = {
```

Comment:
this function can be `private`

Reply:
To be consistent with other validation functions, but happy to change if preferred.

Reply:
yea in general we should keep the code syle the same with existing code in the same file.